features/user/managing_customers/editing_administrator.feature

```diff
@@ -0,0 +1,20 @@
+@managing_customers
+Feature: Editing an administrator
+    In order to change information about an administrator
+    As an Administrator
+    I want to be able to edit my information
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And there is an administrator "Gareth Bale" identified by an email "bale@example.com" and a password "bale11"
+        And I am logged in as "bale@example.com"
+
+    @ui @javascript
```

Comment:
Because in admin panel we have dropdown menu which is a little problem, when I want to press logout button.